### DISCUSSION TITLE
Currently looking at 20 CMS plus WP. Hoping SonicJS will be it!

### DISCUSSION THREAD

**Discussion by easaw** [SonicJs-Org/sonicjs#45] [General]
I want to use a Headless CMS for apps, but I also need a good admin area with control and granular user roles.

Even though other, light CMS (and beasts like WP) are well established, I'm hoping to find something that covers everything I need, but that isn't as bulky (and frankly as insecure) as some others *cough-wordpress-cough*.

I'm currently creating a massive feature comparison chart for all 21 CMS, I'll put a link here when done.

So, thank you Douglas! Let's hope this can really become something amazing!

**Comment by lane711** [comment]
Thanks for you feedback @easaw !

FYI I just pushed a major changes to master - SonicJs is now 100% GraphQL based!
